contentcuration/contentcuration/frontend/administration/views/AdministrationIndex.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+
+  <VApp>
+    <AppBar>
+      <template #tabs show-arrows>
+        <v-layout justify-start>
+          <VTab :to="channelsLink">
+            Channels
+          </VTab>
+          <VTab :to="usersLink">
+            Users
+          </VTab>
+        </v-layout>
+      </template>
+    </AppBar>
+    <VContent>
+      <VContainer fluid>
+        <router-view />
+      </VContainer>
+    </VContent>
+    <GlobalSnackbar />
+  </VApp>
+
+</template>
+
+
+<script>
+
+  import { channelsLink, usersLink } from '../router';
+  import AppBar from 'shared/views/AppBar';
+  import GlobalSnackbar from 'shared/views/GlobalSnackbar';
+
+  export default {
+    name: 'AdministrationIndex',
+    components: {
+      AppBar,
+      GlobalSnackbar,
+    },
+    computed: {
+      channelsLink,
+      usersLink,
+    },
+  };
+
+</script>
+
+
+<style lang="less">
+
+  h1 {
+    font-size: 24px;
+  }
+
+  @pinnedWidth: 300px;
+
+  #data-table {
```

Comment:
I put it here so that I could share it between table components, since it was so large.  What's a good practice for shared css in vue components?

Reply:
Well if you have shared functionality then perhaps that should be it's own component, with the CSS living there? Otherwise, I would put it into a file and import it into each component's CSS.

Reply:
Awesome! I figured out that less imports are perfect for this.